examples/booleans/booleans.description

```diff
@@ -1,3 +1,3 @@
 // The `boolean` type has two values: `true`, `false`.
 // The `!` operator works on booleans only.  `&&` and `||` operators short-circuit as in C.
-// Usual comparison operators produce boolean values: `== != < > <= >=`.
+// Usual comparison operators producing boolean values are `==`, `!=`, `<`, `>`, `<=`, and `>=`.
```

Comment:
I guess what we want to say here is that the usual comparison operators produce booleans. So maybe we should say

```suggestion
// Usual comparison operators (`==`, `!=`, `<`, `>`, `<=`, and `>=`) produce boolean values.
```